commons/passive/google/google_sleep_classify_event.avsc

```diff
@@ -0,0 +1,13 @@
+{
+  "namespace": "org.radarcns.passive.google",
+  "type": "record",
+  "name": "GoogleSleepClassifyEvent",
+  "doc": "Sleep classification event including the sleep confidence, device motion and ambient light level.",
```

Comment:
From this description, I'm not sure what is being classified, e.g. what a classify event record would tell me. Is it connected to the segment event? If so, it would be helpful to have a common identifier, e.g. `segmentId`.

Reply:
Hi, classified events serve as supporting data for segmented events that are reported at regular intervals. In contrast, segmented events are only reported when there is sufficient confidence that the user is awake. As for the relationship between segmented and classified events, the last segmented event can also be reported whenever we subscribe to sleep data, while classified events are independently reported at fixed intervals. Despite this, is there still a need for the segmentId

Reply:
The sleep detection result depends on sampled device motion and ambient light readings during the past day. For the customized sleep segmentation we are using Sleep Classify events.

Reply:
Yes, the segmentId can be used as sleep segment data, depending on the sleep classification data from the previous day. It works in a way that segmentId will increase every time SleepSegmentEvents are recorded. Would you like this or something different?

Reply:
You don't need to use segmentId in that case, I didn't understand from the description what confidence meant. In the main doc, please do add something like `Sleep classification event that indicates that the user is probably sleeping. Includes sleep confidence, device motion and ambient light level.`

Reply:
Okay